src/04_video_container.js

```diff
@@ -228,12 +242,23 @@ class VideoContainerBase extends paella.DomNode {
 		this.slaveVideoData = null;
 		this.currentMasterVideoData = null;
 		this.currentSlaveVideoData = null;
+		this._maxSyncDelay = 0.3; // #DCE OPC-401
+		this._syncHits = 0;  // #DCE OPC-407
+		this._videoSyncTimeMillis = 5000;
 		this._force = false;
```

Comment:
@ferserc1, We have an issue where after getting out of synch from too much seeking, the secondary can never catch up to the master via synch(). So, next week I'm taking this  _syncHits attribute and using it to test a progressive sync strategy: 
- On the first out of synch iteration: (pause secondary) sync to current-master-audio time plus a short offset (play if was playing)
- On the second out of synch iteration within a short time: (pause secondary) synch to current-master-audio time plus a longer offset (play if was playing)
- On the third out of synch iteration within a short time: do a seek to current time on all videos via videoContainer which will pause all videos, seek on them all, and only play them again when  all have seeked. This will cause the audio to hiccup to the user, but the group seek solves the lagging synch catch-up issue.